letta/server/rest_api/routers/v1/agents.py

```diff
@@ -359,7 +359,7 @@ def update_message(
     return server.update_agent_message(agent_id=agent_id, request=request)
 
 
-@router.post("/{agent_id}/messages", response_model=LettaResponse, operation_id="create_agent_message")
```

Comment:
This apparently should be `None` because it may be a `StreamingResponse` or a `LettaMessage` (`Union[...]` throws an error).

Reply:
isn't this going to mess up the autodocs?

Reply:
Well the autodocs are wrong right now anyways